release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1055,7 +1092,7 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, if a machine config included a compressed file with the `spec.config.storage.files.contents.compression` parameter set to `gzip`, the Machine Config Daemon (MCD) incorrectly wrote the compressed file to disk without extracting it. With this fix, the MCD now extracts a compressed file when the compression parameter is set to `gzip`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970218[*BZ#1970218*])
 
-* Previously, systemd units were cleaned up only when completely removed. As a result, systemd units could not be unmasked by using a machine config because the masks were not being removed unless the systemd unit was completely removed. With this fix, when you configure a systemd unit as `mask: true` in a machine config, any existing masks are removed. As a result, systemd units can be unmasked.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966445[*BZ#1966445*])
+* Previously, systemd units were cleaned up only when completely removed. As a result, systemd units could not be unmasked by using a machine config because the masks were not being removed unless the systemd unit was completely removed. With this fix, when you configure a systemd unit as `mask: true` in a machine config, any existing masks are removed. As a result, systemd units can be unmasked. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966445[*BZ#1966445*])
```

Comment:
```suggestion
* Previously, systemd units were cleaned up only when completely removed. As a result, systemd units could not be unmasked by using a machine config because the masks were not being removed unless the systemd unit was completely removed. With this fix, when you configure a systemd unit as `mask: true` in a machine config, any existing masks are removed. As a result, systemd units can now be unmasked. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966445[*BZ#1966445*])
```
Not a necessary addition, but I like to stress the "then vs now" when talking about bug fixes

Reply:
Updated. Thanks.